ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/get_anta_results.py

```diff
@@ -123,6 +128,26 @@ def get_anta_results(
     )
 
 
+def load_custom_catalogs(catalog_files: list[Path]) -> dict:
+    """Load the custom ANTA catalogs from the provided files and merge them together.
```

Comment:
would the order be important?

Reply:
No, if we have duplicate tests (same inputs) ANTA will remove them.